app/jobs/scraper/new_york_job.rb

```diff
@@ -0,0 +1,42 @@
+class Scraper::NewYorkJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    url = "https://dol.ny.gov/apprenticeship/apprenticeship-trades"
+    uri = URI.parse(url)
+    response = Net::HTTP.get_response(uri)
+    html = response.body
+    doc = Nokogiri::HTML(html)
+
+    table_and_nav = doc.css("div.wysiwyg--field-webny-wysiwyg-body")
+    table = table_and_nav.css("tbody")[0]
```

Comment:
Are there multiple tables on the page? If not, can we just directly get the `table` from the `tbody` rather than starting with the `wysiwyg--...` class? That class seems like that could change.

Reply:
There are 3 tables and we only need the first one so I changed it to just grab the first one without querying for the wrapper div first